lxd/storage/backend_lxd.go

```diff
@@ -2040,8 +2040,11 @@ func (b *lxdBackend) EnsureImage(fingerprint string, op *operations.Operation) e
 	// Derive content type from image type. Image types are not the same as instance types, so don't use
 	// instance type constants for comparison.
 	contentType := drivers.ContentTypeFS
+	dbContentType := db.StoragePoolVolumeContentTypeNameFS
+
 	if image.Type == "virtual-machine" {
 		contentType = drivers.ContentTypeBlock
```

Comment:
Having a function which converts from contentType to dbContentType and calling that later in the DB call would likely be cleaner.

Reply:
@stgraber @monstermunchkin it would be similar to these, and could go with them. https://github.com/lxc/lxd/blob/master/lxd/storage/utils.go#L79